### PR TITLE
simplifications in configuration. shared javaexec arguments

### PR DESCRIPTION
**Pull request by mklaehn**
reduced duplication of shared arguments like "-oAuthFile", "MyOAuth.properties" which now reside in one place

### CODE REVIEW

File: 2d/src/main/java/org/tweetwallfx/twod/TagTweets.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public void stop() {
         }
     }
 
-    private class TweetsUpdateTask extends Task<Void> {
+    private static class TweetsUpdateTask extends Task<Void> {
```
Comment (svenreimers) [inline 2d/src/main/java/org/tweetwallfx/twod/TagTweets.java]: Does not belong in this change

**Comment by svenreimers** [comment]
Please create issue first

**Comment by mklaehn** [comment]
cancel PR
